lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -398,6 +398,8 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
 
     variant = to_track_variant(rid)
 
+    Membrane.Logger.info("New incoming WebRTC track #{track_id} with SSRC #{inspect(ssrc)}")
```

Comment:
We have already similar log in the engine module https://github.com/jellyfish-dev/membrane_rtc_engine/blob/1620023e0a1328b98a979690e13059afa8690248/lib/membrane_rtc_engine/engine.ex#L711 so I would put this one on debug